core/src/main/java/org/elasticsearch/search/aggregations/pipeline/movavg/models/HoltWintersModel.java

```diff
@@ -157,11 +177,40 @@ public HoltWintersModel(double alpha, double beta, double gamma, int period, Sea
         this.padding = seasonalityType.equals(SeasonalityType.MULTIPLICATIVE) && pad ? 0.0000000001 : 0;
     }
 
+    @Override
+    public boolean minimizeByDefault() {
+        return true;
+    }
+
+    @Override
+    public boolean canBeMinimized() {
+        return true;
+    }
 
     @Override
-    public boolean hasValue(int windowLength) {
+    public MovAvgModel neighboringModel() {
+        double newValue = Math.random();
+        switch ((int) (Math.random() * 3)) {
+            case 0:
+                return new HoltWintersModel(newValue, beta, gamma, period, seasonalityType, pad);
+            case 1:
+                return new HoltWintersModel(alpha, newValue, gamma, period, seasonalityType, pad);
+            case 2:
+                return new HoltWintersModel(alpha, beta, newValue, period, seasonalityType, pad);
+            default:
+                return new HoltWintersModel(newValue, beta, gamma, period, seasonalityType, pad); // This should never technically happen...
```

Comment:
worth adding an assert(false) here (maybe with a message) so we can assure it never happens?